action.yml

```diff
@@ -17,6 +17,9 @@ inputs:
   force-push:
     description: 'Force push to Platform.sh'
     required: false
+  instance-name:
```

Comment:
How about calling this `environment-name` instead? We'll be using the same vocab as platform.sh then.

Reply:
Updated.